src/transformers/onnx/convert.py

```diff
@@ -261,9 +275,15 @@ def validate_model_outputs(
     # TODO: generate inputs with a different batch_size and seq_len that was used for conversion to properly test
     # dynamic input shapes.
     if issubclass(type(reference_model), PreTrainedModel):
-        reference_model_inputs = config.generate_dummy_inputs(tokenizer, framework=TensorType.PYTORCH)
+        reference_model_inputs = config.generate_dummy_inputs(
+            preprocessor,
+            tokenizer=tokenizer,
+            framework=TensorType.PYTORCH,
```

Comment:
Fits in one line
```suggestion
            preprocessor, tokenizer=tokenizer, framework=TensorType.PYTORCH
```

Reply:
Indeed it fits in one line, but curiously `make style` formats it back to the multiline case (even though it's shorter than the TensorFlow line beneath it 🤔 )

Reply:
That's because you did not commit my suggestion but rewrote it in your file with a trailing comma ;-)
You should commit suggestions as it gives the co-autorship to the users who wrote them.